source/postprocess/crystal_preferred_orientation.cc

```diff
@@ -208,7 +210,14 @@ namespace aspect
       std::stringstream string_stream_content_raw;
       std::stringstream string_stream_content_draw_volume_weighting;
 
-      string_stream_master << "id x y" << (dim == 3 ? " z" : "") << " olivine_deformation_type" << std::endl;
+      string_stream_master << "id x y" << (dim == 3 ? " z" : "") << " olivine_deformation_type"
```

Comment:
The programming community is trying to avoid the word "master". I know this is pre-existing, but could you try to find a different name for this variable?

Reply:
It's ok if you wanted to do this as a follow-up.